src/python/pants/java/nailgun_client.py

```diff
@@ -17,6 +17,7 @@
 from pants.java.nailgun_protocol import ChunkType, NailgunProtocol
 from pants.util.osutil import safe_kill
 from pants.util.socket import RecvBufferedSocket
+from pants.util.strutil import ensure_binary
 
 
```

Comment:
This was failing when running with `./pants2`, and I'm not sure why, but it seems like a correct edit to make regardless.